handout/components/app_structure/passing_data_into_components.md

```diff
@@ -1,6 +1,8 @@
 # Passing Data into a Component
 
-The `inputs` attribute defines a set of parameters that can be passed down from the component's parent. For example, we can modify the `Hello` component so that `name` can be configured by the parent.
+There are two ways to pass data into a component, with 'property binding' and 'event binding'. Data flows top-down from parent into children, while events flow bottom-up from child to parent. Events works just like they do in regular HTML DOM based events and are cancelable.
```

Comment:
`EventEmitter` emissions are cancellable? As far as I know, this is false. If you were to implement some kind of cancellation policy I think you would have to do it manually. (But maybe there is a way to do it and I just don't know? If so, add more detail to this paragraph on how to cancel an `EventEmitter` event.)